rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -793,3 +794,46 @@ def test_aave_v3_repay_scroll(database, scroll_inquirer, scroll_accounts) -> Non
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x35E0091D67B5e213db857F605c2047cA29A8800d']])
+def test_non_aave_tx(database, ethereum_inquirer, ethereum_accounts) -> None:
+    """Test that the non-aave transactions happened through aave flash loans are not decoded
```

Comment:
Just for the comment line: what happened in that transaction is not an "aave flash loan", but a flash loan taken by Spark. There also other protocols providing flash loans, like Balancer, and others that I don't know. So this test case sounds valid for all the flash loan types. Also, that transaction is useful because is containing a Lido transaction, with stETH tokens generated from NULL address, as the aave variable debts, so this PR hardens the check also for the generated tokens from NULL addresses.